src/hooks/useList.js

```diff
@@ -29,7 +30,23 @@ export default function useList() {
     visibilityUndoButton,
   }
 
-  function addListItem(title, isChecked = false) {
+  function handleInputChange(event) {
```

Comment:
In some functions you just write ``e``and sometimes ``event``

Reply:
Changed it. Thanks.